app/components/user-skills-input.js

```diff
@@ -1,16 +1,28 @@
+import { EKMixin, keyDown, getCode } from 'ember-keyboard';
 import Ember from 'ember';
 
 const {
   Component,
   computed,
-  computed: { alias, and, equal, not, notEmpty },
+  computed: { alias, and, not, notEmpty },
   inject: { service },
   isEmpty,
   run: { once },
-  observer
+  observer,
+  on,
+  TextField
 } = Ember;
 
-export default Component.extend({
+Ember.TextField.extend(EKMixin, {
```

Comment:
@sbatson5 Oh yeah, for some reason, the tests would fail if I deleted the `Ember.` prefix and used the destructured variable instead. Really bizarre. I added the `Ember.` prefix back to make the tests pass, but I forgot to delete the destructuring part.